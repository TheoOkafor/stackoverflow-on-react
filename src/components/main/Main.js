import React, { Component } from 'react';
import PropTypes from 'prop-types';

class Main extends Component {
  render() {
    return (
      <div className="sd-margin-12 d-margin-7 margin-top-15">
        <div className="row-container" id="main-container">
          { this.props.children }

        </div>
      </div>
    );
  }
}

Main.propTypes = {
  children: PropTypes.element,
}

export default Main;
