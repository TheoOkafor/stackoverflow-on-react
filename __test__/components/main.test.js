import React from 'react';
import Main from '../../src/components/main/Main';
import renderer from 'react-test-renderer';

test('Test that the Home displays', () => {
  const component = renderer.create(
    <Main />
  );
  expect(component).toMatchSnapshot();
});
