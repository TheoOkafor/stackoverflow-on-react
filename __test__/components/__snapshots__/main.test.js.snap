// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test that the Home displays 1`] = `
<div
  className="sd-margin-12 d-margin-7 margin-top-15"
>
  <div
    className="row-container"
    id="main-container"
  />
</div>
`;
